docs/DeveloperGuide.md

```diff
@@ -152,110 +153,6 @@ The `Storage` component,
 
 Classes used by multiple components are in the `seedu.netconnect.commons` package.
 
---------------------------------------------------------------------------------------------------------------------
-
-## **Implementation**
```

Comment:
should we retain the implementation section? looks like something that we will/need to add to while we implement our features